concrete/views/panels/add.php

```diff
@@ -219,6 +219,9 @@ class="ccm-panel-add-clipboard-block-item"
                 }
                 ?>
                 <script>
+                //Hides grid view switcher
+                $('.ccm-panel-header-list-grid-view-switcher').hide();
```

Comment:
Is there code in the other panels to show this switcher?

Reply:
The switcher shows by default in other panels, not really sure if the others will use the switcher.